testsuite/MDAnalysisTests/analysis/test_msd.py

```diff
@@ -41,14 +41,29 @@ def SELECTION():
 
 
 @pytest.fixture(scope='module')
-def NSTEP():
-    nstep = 5000
-    return nstep
+def u():
+    return mda.Universe(PSF, DCD)
+
+
+@block_import('tidynamics')
+def test_notidynamics(u, SELECTION):
+    # Have to avoid importing until here as otherwise tidynamics will already
+    # be imported
+    from MDAnalysis.analysis.msd import EinsteinMSD as MSD
```

Comment:
I'm not super happy about how messy this looks, but it's the cleanest way I could think of testing this without having the global import override the presence of tidynamics..